src/arrays.h

```diff
@@ -30,9 +30,10 @@ extern char gtk_str[MAX_STRLEN];
 extern char icon_str[MAX_STRLEN];
 extern char font_str[MAX_STRLEN];
 
-extern char host_color[10];
+extern char host_color[MAX_STRLEN];
```

Comment:
Defined as `host_color[MAX_STRLEN]` [here](https://github.com/woodruffw/screenfetch-c/pull/32/files#diff-a99cb0be746bdda6a873087ba1a7795cR33), so I changed it.